src/routes/aggregators.rs

```diff
@@ -62,6 +33,24 @@ impl FromConn for Aggregator {
     }
 }
 
+impl Permissions for Aggregator {
+    fn allow_read(&self, actor: &crate::PermissionsActor) -> bool {
+        actor.is_admin()
+            || match &self.account_id {
+                Some(account_id) => actor.account_ids().contains(account_id),
+                None => true,
```

Comment:
This is covering the case where we have a global aggregator, correct? If the account ID is unspecified, then it's a global aggregator, and anyone should be able to read it?

Reply:
correct, that's exactly what this code is saying

Reply:
I suppose a clarity improvement would be to have a method on `Aggregator`.
```
fn is_global_aggregator(&self) -> bool {
    self.account_id.is_none()
}
```
I don't feel strongly about this, can take it or leave it.

Reply:
I think that would make this code more complex, because it would be `actor.is_admin() || self.is_global_aggregator() || self.account_id.expect("this is ok because we already checked if it's a global aggregator")`

Reply:
My intent wouldn't be to necessarily shorten the code, but to make it more explicit that a global aggregator has special behavior. This code is straightforward enough though that it isn't worth it. I appreciate the consideration though.